<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerGroupWidget</class>
 <widget class="QWidget" name="LayerGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>61</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#group_dimensions_label_{
	color: white;
	font: bold;
}
QWidget#LayerGroupWidget{
	background-color: rgba( 0, 0, 0, 0 );
}

QToolButton#group_visibility_button_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}
QToolButton#group_visibility_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}
QToolButton#group_iso_visibility_button_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}
QToolButton#group_iso_visibility_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}

QFrame#group_frame_{
	background-color: rgba(0, 0, 0, 0);
	border: none;
}

QToolButton#open_button_{
	background-color: rgba(0, 0, 0, 0);
	border-radius: 4px;
}

QToolButton#open_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}

QWidget#group_header_{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(0, 0, 0, 0), stop:0.45 rgba(0, 0, 0, 30), stop:1 rgba(0, 0, 0, 25));
    border-radius: 3px;
}

QWidget#group_background_{
	background-color: rgb(232, 145, 32);
    border-radius: 3px;
	border: 1px solid rgb(90, 90, 90);
}

QWidget#base_{
	background-color: rgb( 90, 90, 90 );
	border-radius:4px;
	border: 1px solid rgb(60, 60, 60);
	color: rgb(90, 90, 90);
}

QWidget#tools_and_layers_widget_{
	background-color: rgba( 0, 0, 0, 0 );
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="base_" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QWidget" name="group_background_" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>21</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="group_header_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>215</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>21</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="open_button_">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click here to show or hide the contents of this group</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/RightArrowWhite.png</normaloff>
                <normalon>:/Images/DownArrowWhite.png</normalon>:/Images/RightArrowWhite.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QtUtils::QtTristateToolButton" name="group_visibility_button_" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Turn on or off the group's visibility</string>
              </property>
              <property name="text" stdset="0">
               <string/>
              </property>
              <property name="icon" stdset="0">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/VisibleOff.png</normaloff>
                <normalon>:/Images/VisibleWhite.png</normalon>:/Images/VisibleOff.png</iconset>
              </property>
              <property name="iconSize" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="group_iso_visibility_button_">
              <property name="minimumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>21</width>
                <height>21</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click to hide/show computed iso surfaces for this group</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/IsosurfaceVisibleOff.png</normaloff>
                <normalon>:/Images/IsosurfaceVisibleOn.png</normalon>:/Images/IsosurfaceVisibleOff.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="group_dimensions_label_">
              <property name="toolTip">
               <string>Dimenions of the layers in this group</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tools_and_layers_widget_" native="true">
        <layout class="QVBoxLayout" name="tools_and_layers_layout_">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="group_frame_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QVBoxLayout" name="group_frame_layout_">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fake_widget_">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="group_dummy_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QtUtils::QtTristateToolButton</class>
   <extends>QWidget</extends>
   <header location="global">QtUtils/Widgets/QtTristateToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/QtResources.qrc"/>
 </resources>
 <connections/>
</ui>
